packages/node/src/lib/subscribers/subscribers.ts

```diff
@@ -110,27 +129,53 @@ export class Subscribers extends WithHttp implements ISubscribers {
     );
   }
 
+  /**
+   * deprecated use markInAppMessageAs instead
+   */
   async markMessageSeen(subscriberId: string, messageId: string) {
-    return await this.http.post(
+    return await this.novu.post(
       `/subscribers/${subscriberId}/messages/markAs`,
       { messageId, mark: { seen: true } }
     );
   }
 
+  /**
+   * deprecated use markInAppMessageAs instead
+   */
   async markMessageRead(subscriberId: string, messageId: string) {
-    return await this.http.post(
+    return await this.novu.post(
       `/subscribers/${subscriberId}/messages/markAs`,
       { messageId, mark: { read: true } }
     );
   }
 
+  async markInAppMessageAs(
```

Comment:
I feel markMessageAs would fit better with our overall use. What do you think? Is there a specific reason you want to call it like that?

Reply:
having InApp in method name signifies that this method can mark only in-app messages.
Some users got confused that it can mark other channel messages as well.

Reply:
This method still can mark other channels as well, so it could confuse the user even more. 
This method is not intended to be only for in-app it should allow potentially mark other channels as well. so users should be aware of the message ids and not mark other channels. 
Although we could allow passing channel type to help users to guard the mark.

Reply:
Shall we update the API for this?